### ISSUE TITLE
Question 18 of 144: The government should run the healthcare insurance industry

### ISSUE THREAD

**Issue by RMcD94** [sixtriangles/sixtriangles.github.io#8]
My country does not have a healthcare insurance industry

**Comment by RMcD94** [comment]
Question 20 of 144: There should be no billionaires

No currency is specified. Do Indonesian or Iranian billionaires really have the same impact as British ones?

**Comment by RMcD94** [comment]
Question 21 of 144: Excessive government intervention hurts the economy

Excessive is defined as too much. Any answer that disagrees with this is factually false

**Comment by RMcD94** [comment]
Question 24 of 144: Speculations and facts which cannot be confirmed for certain should never be published

Nothing can be confirmed for certain.

**Comment by RMcD94** [comment]
Question 51 of 144: Those who take credit for the work of others should lose everything

Have their children murdered because you pretended someone's homework was yours?

**Comment by RMcD94** [comment]
Question 83 of 144: The government is larger than it needs to be

"needs"

So not what the optimal is, not what you want, what is necessary for it be called a government. Pointless question every government is larger than strictly necessary.

**Comment by RMcD94** [comment]
Question 92 of 144: It is a waste of our soldiers and resources to topple other governments, regardless of what they do

"regardless" so if they are nuking your cities? again you make the answer have no other choice with framing of question

**Comment by RMcD94** [comment]
"Everything that can be left to the free market should be"

If someone is a commie and thinks nothing can be left to the free market they could also agree to this question

**Comment by RMcD94** [comment]
Question 103 of 144: Automation has the potential to give significant benefits to humanity

"potential" again, as long as it has 0.0001% chance of being true then it's always agreew

**Comment by sixtriangles** [comment]
The government should run the healthcare insurance industry- This means "The government should pay for everyone's healthcare"
No currency is specified. Do Indonesian or Iranian billionaires really have the same impact as British ones? - This is a fair point. I'll consider adding a wordier, more international answer.
Excessive is defined as too much. Any answer that disagrees with this is factually false - I can change this, though this is designed to be strong, to filter out, for example, communists, who think that the government should run the entire economy. Then, there's no such thing as excessive government intervention.
Nothing can be confirmed for certain. - That's an opinion. You think that it's not confirmed for certain that Donald Trump is the president of the United States? Or you think there's a tiny chance that there's no protests going on in the US right now?
Have their children murdered because you pretended someone's homework was yours? - Then put strongly disagree, though it wasn't implying lives, more things like credit, property, etc.
So not what the optimal is, not what you want, what is necessary for it be called a government. Pointless question every government is larger than strictly necessary. - You're assuming the question is The government is larger than it needs to be (to be a government), when it's saying The government is larger than it needs to be (to be a good government). Perhaps I could reword this as "The government should be smaller", but I think most people can figure out that's what it means.
"regardless" so if they are nuking your cities? again you make the answer have no other choice with framing of question - we don't say we toppled the government of Nazi Germany, we say we defeated them in war. Toppling governments is typically considered an aggressive act.
If someone is a commie and thinks nothing can be left to the free market they could also agree to this question - I mean, they could, but I very much doubt any of them would, since they know what it's meaning. Perhaps a better way to think of this is, imagine a politician was saying each of these statements. Would you support or be against this politician? If a politician says 
"Everything that can be left to the free market should be", is a communist gonna support him? Very very likely not.
"potential" again, as long as it has 0.0001% chance of being true then it's always agreew - Good, congratulations, you're not afraid of technology. 

If it helps you, think of these as politicians saying these, and do you agree with them or think they're an idiot? Of course they're exaggerated a bit, that's what politicians do.
